handsontable/src/editors/autocompleteEditor/autocompleteEditor.js

```diff
@@ -257,6 +257,26 @@ export class AutocompleteEditor extends HandsontableEditor {
     this.hot.view.render();
   }
 
+  /**
+   * Finishes editing and start saving or restoring process for editing cell.
+   *
+   * @param {boolean} restoreOriginalValue If true, then closes editor without saving value from the editor into a cell.
+   * @param {boolean} ctrlDown If true, then saveValue will save editor's value to each cell in the last selected range.
+   * @param {Function} callback The callback function, fired after editor closing.
+   */
+  finishEditing(restoreOriginalValue, ctrlDown, callback) {
+    if (this.isOpened()) {
+      const lastSelectedRange = this.hot.getSelectedRangeLast();
```

Comment:
The value can be edited in every active selection layer, not just the last one.
```suggestion
      const lastSelectedRange = this.hot.getSelectedRangeActive();
```